tcp/flow.ml

```diff
@@ -14,7 +14,12 @@
  * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  *)
 
-open Lwt
+let (>>=) = Lwt.(>>=)
```

Comment:
I never know in which version of `Lwt` the `Infix` module appeared. Thanks for the suggestion, I'll use that from now on.